### PR TITLE
Feature/pwa setup

### LINKED ISSUE #34: As a user, I should be able to download this app to my phone/computer from the website.

### PR DESCRIPTION
**Pull request by AshleyOh-bit**
## Description
- Registered serviceWorker in index.js
- Edited manifest to reflect app details
- Updated logos to be new favicon

## Concerns

-  Any bugs still present?
No - to accurately test pwa on local host, use npm run build in terminal and follow subsequent instructions. Then test with Lighthouse through Chrome.
-  Next steps?
Apply selected features!

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactor

## Checklist:
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] No tests added for this feature

##Closed Issues
closes #34

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -20,6 +21,8 @@
   ],
   "start_url": ".",
   "display": "standalone",
+  "orientation": "portrait",
   "theme_color": "#000000",
```
Comment (tylrs) [inline public/manifest.json:25]: I think we should change the theme color and maybe the background color to something from our palette

Comment (AshleyOh-bit) [reply public/manifest.json:25]: Good idea. Updating that now!

**Review by NoahZinter — APPROVED** [review]

File: src/index.js [hunk 2]
```diff
@@ -15,4 +15,4 @@ ReactDOM.render(
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://cra.link/PWA
-serviceWorkerRegistration.unregister();
+serviceWorkerRegistration.register();
```
Comment (NoahZinter) [inline src/index.js:18]: Do these service workers work similarly to background workers in rails?

File: public/manifest.json [hunk 3]
```diff
@@ -20,6 +21,8 @@
   ],
   "start_url": ".",
   "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "orientation": "portrait",
+  "theme_color": "#0E1525",
```
Comment (tylrs) [inline public/manifest.json:25]: Thank you for changing this

**Review by tylrs — APPROVED** [review]

**Pull request merged**